docs/pages/tokens/typography.tsx

```diff
@@ -35,12 +131,38 @@ export default function TokensTypographyPage() {
 					</p>
 					<p>We have individual tokens for...</p>
 					<ul>
-						<li>font</li>
 						<li>fontSize</li>
-						<li>fontWeight</li>
 						<li>lineHeight</li>
+						<li>font</li>
+						<li>fontWeight</li>
 					</ul>
-					<TypographyChart />
+
+					<h2>fontSize</h2>
+					<FontSizeTable />
+					<h2>lineHeight</h2>
+					<LineHeightChart />
+
+					<h2>font</h2>
+					<p>A collection of tokens for font family groupings.</p>
+					<FontFamilyTable />
+
+					<h2>fontWeight</h2>
+					<div className={proseBlockClassname}>
```

Comment:
I think this would look better as a table with `Token | Value | Description`, similar to the other tables on this page.

Reply:
For fontWeight, the token and value are the same.
From a stylistic POV I see what you mean